lib/commands/ls.js

```diff
@@ -23,5 +23,6 @@ class LsCommand extends Command {
 
 LsCommand.description = 'View running ghost processes';
 LsCommand.global = true;
+LsCommand.aliases = ['status'];
```

Comment:
Haha I just followed that MR and didn't see the follow on commit!

Will change this over to the new syntax now. Vim is highlighting `status` as a var or command or some sort, I assume I have to quote it?